borg/helpers.py

```diff
@@ -8,6 +8,12 @@
 import os
 import pwd
 import re
+try:
+    from shutil import get_terminal_size
+except ImportError:
+    def get_terminal_size(fallback=(80, 24)):
+        TerminalSize = namedtuple('TerminalSize', ['columns', 'lines'])
+        return TerminalSize(int(os.environ.get('COLUMNS', int(fallback[0]))), int(os.environ.get('LINES', int(fallback[1]))))
```

Comment:
hmm, why do you cast the fallback value to int (with that inner int())?
if it is used, it will be cast to int by the outer int().